docs/data-imports.md

```diff
@@ -0,0 +1,52 @@
+# Data Imports
+
+The data imports pick up where the ETL leaves off by copying and importing the
+CSV files into the Postgresql database. This document describes that process in
+more detail and also some of the schema choices that were made to optimize the
+imports for speed.
+
+The examples below will use Firefox on Android as the example, as it is a Glean
+based product and will be the example for future products in GLAM going forward.
+
+## Schema
+
+The Firefox for Desktop (aka fenix) product has 2 primary tables assocated with
+its data. The aggregation table and the materialized view on top of that data.
+
+The aggregation table exist so imports can be as fast as possible. It has
+minimal indexes and constraints so data can import quickly. The only index is
+the primary key. The only constraint is a unique constraint comprised of the
```

Comment:
> The only index is the primary key.
So the only index for the aggregation table is the auto-generated id right?

Reply:
Right. I can try to clarify that some there.